src/ImageLoaders/Pef/Pef.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Inject the following XML right after the `<Project>` opening tag.
```
  <Import Project="$(ProjectDir)../../Drivers/CommonBuildProperties.items"/>
```
This may trigger nullable warnings and other compiler issues.